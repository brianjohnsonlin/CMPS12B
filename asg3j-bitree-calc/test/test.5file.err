./pbitreecalc.perl: data5.in: 2: syntax error
./pbitreecalc.perl: data5.in: 3: syntax error
./pbitreecalc.perl: data5.in: 4: syntax error
./pbitreecalc.perl: data5.in: 5: syntax error
./pbitreecalc.perl: data5.in: 6: syntax error
./pbitreecalc.perl: data5.in: 7: syntax error
./pbitreecalc.perl: data5.in: 8: syntax error
./pbitreecalc.perl: data5.in: 9: syntax error
./pbitreecalc.perl: data5.in: 10: syntax error
