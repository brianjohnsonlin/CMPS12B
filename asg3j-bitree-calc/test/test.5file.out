**** ## $Id: data5.in,v 1.1 2013-10-16 12:50:14-07 - - $ ##
**** hello there
**** h=hello
**** y=hello++++++++++++++
**** z=hello!@#$%^&*()_+|~
**** r:foobar
**** u¿
**** x=aa×bb÷*
**** 3.1415926535
**** p:MCMLXIV
**** FINAL SYMBOL TABLE ****
